manifest.md

```diff
@@ -42,6 +42,12 @@ Unlike the [image index](image-index.md), which contains information about a set
 
         If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
 
+    To set an effectively NULL or SCRATCH config and maintain portability the following is considered GUIDANCE.
+    While a 0 `size` may be preferable, practice has shown that not to be ubiquitious supported.
+    Instead, the blob payload can be the most minimal content that is still valid JSON: `{}` (size of 2).
```

Comment:
*valid JSON *object*. I assume an object is needed because `0` fails to unmarshal into a struct in most languages?